### ISSUE TITLE
German BA.2/BA.1/BA.2 recombinant with S:K147E, S:R346K, S:460K, S:493 reversion, S:H1101Y [55 seq as of 2022-07-22]

### ISSUE THREAD

**Issue by JosetteSchoenma** [cov-lineages/pango-designation#823]
@FedeGueli spotted this one and mentioned it in #773
 **Description:**
**Recombinant between:** BA.2/BA.1/BA.2
**Earliest sequence:** 1st of June 2022
**Most recent sequence:** 23th of June 2022
**Country:** Germany
**Likely breakpoint:** between 13195 and 15240 and between 21618 and 21762 [S:27 to S:67]
Private Mutations: C4927T, T5386G (like BA.1), C7834T, C12049T, A220001G/S:K147E, G22599A/S:R346K, G22992/S:S477N, C24863T/S:H1101Y, A27507G
**Cov-Spectrum query:** https://cov-spectrum.org/explore/World/AllSamples/Past3M/variants?nucMutations=C4321T%2CT5386G%2CC9344T%2CA18163G%2CG21987A%2CC24863T%2CA27507G%2CC12880T&variantQuery1=Nextcladepangolineage%3ABA.5&
**Genomes:** 
EPI_ISL_13569221
EPI_ISL_13385893
EPI_ISL_13344896
EPI_ISL_13569385
EPI_ISL_13567682
EPI_ISL_13387015
EPI_ISL_13570913
EPI_ISL_13393217
EPI_ISL_13269776
EPI_ISL_13382835
EPI_ISL_13382506
EPI_ISL_13382561
EPI_ISL_13380484
EPI_ISL_13269235
EPI_ISL_13387937
EPI_ISL_13269123
**Evidence:**
Nextclade output:
Red is BA.1
![image](https://user-images.githubusercontent.com/94831035/177489569-a214a6b2-68f0-4cef-a8c9-daa7105d01d1.png)

Sc2rf [@corneliusroemer replaced Delta/Omicron sc2rf with relevant BA.1/BA.2]:
<img width="2195" alt="image" src="https://user-images.githubusercontent.com/25161793/180064866-cf0d88ca-9c77-47ed-a030-d0419afc3681.png">

Usher Tree:
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_33c33_216ac0.json?branchLabel=Spike%20mutations&label=nuc%20mutations:C15240T
![image](https://user-images.githubusercontent.com/94831035/177490085-2e95f3ab-b17f-44ec-b131-d44e06d978ae.png)

Original discussion in issue #773 with @FedeGueli and @Silcn
![image](https://user-images.githubusercontent.com/94831035/177490458-06c89d7f-fe29-47f8-9d54-0373b5834a97.png)
![image](https://user-images.githubusercontent.com/94831035/177490565-0c4834cd-7bcc-46a4-b738-13b88e7dc6ac.png)

**Comment by FedeGueli** [comment]
Thx for analysing these sequences! great job.
cc @silcn

**Comment by FedeGueli** [comment]
Good point thx @silcn

**Comment by shay671** [comment]
In Israel we got a sample (will soon be uploaded to GISAID) of a sample clustering with this branch of a passenger coming from Greece (no contact to Germany)

**Comment by FedeGueli** [comment]
Thx @shay671 to me it is still contested if it is a plain recombinant , a recombinant in a chronic case or something coming from the entourage of original Omicron source.
The @silcn argument on Germany is weakened by the fact that BA.5+S:346I was barely sequenced in South Africa and then popped up and emerged fastly in Germany, represwnting a brand new branch of BA.5 distinct from the main designated or even proposed ones

More:
And i  was just reading casually this paper yesterday :

"Potential evidence of a single recombination event involving BA.1, BA.2 and BA3 was identified by 3SEQ (P = 0.03), GARD (delta c-AIC = 20) and RDP5 (GENECONV P = 0.027; RDP P = 0.006) within the NTD encoding region of spike. The most likely breakpoint locations for this recombination event were 21690 for the 5′ breakpoint (high likelihood interval between 15716 and 21761) and 22198 for the 3′ breakpoint (high likelihood interval between 22197 and 22774)."  (Tulio de Oliveira et Al:
https://www.nature.com/articles/s41586-022-04411-y)
the original BA.1/BA.2/BA.3 seem to overlap with the breakpoints f this one described here. Sincerely i dunno know if significant or not , maybe someone else could take a 2nd look at this.

**Comment by silcn** [comment]
@FedeGueli BA.5+S:346I doesn't come directly from the Omicron source though. There have been at least 5 (maybe more, see #455, #561) separate emergence from the Omicron source and in every case most of the early sequences came from South Africa.

**Comment by JosetteSchoenma** [comment]
Thanks for all the comments! It have me some extra insight.
I think it is more complicated as a normal recombinant with 2 breaking points. I missed the 69/70 del, since it is not in Sc2rf, but that would make it 2 BA.1 mutation clusters instead of one. 
Together with the T5386G BA.1 mutation, which is shown by Sc2rf, and I did notice, but didn't highlight as BA.1 (just edited it in the first part of the issue) and the 493 reversion (A23040A) that really makes it very complicated.

To summarize the nucleotides:
1-4321 BA.2
T5386G BA.1
8393-13195 BA.2
15240-21618 BA.1
C21762C BA.2
21765-21770 BA.1 (69/70del)
21846-22882 BA.2
A23040A S:493Q reversion, no BA.1/2
23048-29510 BA.2
Plus of course several private mutations.

**Comment by silcn** [comment]
@JosetteSchoenma also possible that 69/70del occurred independently rather than being BA.1-derived. Wouldn't be too surprising given the other additional spike mutations.

**Comment by FedeGueli** [comment]
Thanks vm Josette! I suggest to edit your main comment so it would be clearer for everyone!

**Comment by FedeGueli** [comment]
@silcn yes that is possible, but in a landscape of multiple breakpoints and a key reversion for other Omicron lineages i dont know how much we can be sure of that or not.

**Comment by shay671** [comment]
As I mentioned yesterday, a first Israeli sample (traveler from Greece) :

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_23033_861f50.json?c=userOrOld&label=nuc%20mutations:T15714C,T17410C,T21618C,A22001G,G22599A,C24863T,A27507G

**Comment by UlrichElling** [comment]
We have two cases in Austria as well, currently uploading to Github.

**Comment by FedeGueli** [comment]
@UlrichElling thx a lot! 

@chrisruis @corneliusroemer @AngieHinrichs @InfrPopGen in my view this has to be granted an early designation, whatever as recombinant or something else, still very different from everything else circulating (or better saying that it has a unique mix of mutations actually on the rise) .

**Comment by UlrichElling** [comment]
What I find remarkable about this variant is that it appears to be convergent evolution with BA.2.75  (147, 460, 493) and with regards to the 493 reversion even BA.5. It seems obvious the virus has identified the next surfaces to optimise.

**Comment by shay671** [comment]
@UlrichElling  
This is why it might be a combination of recombination+saltation in a chronic patient. 

The mutations converged in this one are indeed more related to variants that arose outside SA and the parallel BA.1-5.

And the recombination is much cleaner than could be seen in cases where presumably a wide range of variants been co-evolved like might be in the original chronic patient ( like BA.3 probably coming from the same body where BA.1 and BA.2 evolved and XB probably coming from the same body were B.1.631 and B.1.634 evolved).

**Comment by shay671** [comment]
If anyone needs -here is a comparison of the mutational profile of this variant , BA.1 and BA.2
[PDI 823 .xlsx](https://github.com/cov-lineages/pango-designation/files/9084781/PDI.823.xlsx)

**Comment by FedeGueli** [comment]
Thx @shay671 !

**Comment by shay671** [comment]
For CovSpectrum query i would replace to :
[4-of:C4927T,C7834T,C12049T,A22001G,G22599A,T22942G,C24863T,A27507G]
searching 4 out of the 8 unique mutations in this lineage (not present in BA.1 or BA.2).

**Comment by JosetteSchoenma** [comment]
Here's a link for @shay671 's query.
https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?variantQuery=%5B4-of%3AC4927T%2CC7834T%2CC12049T%2CA22001G%2CG22599A%2CT22942G%2CC24863T%2CA27507G%5D&

**Comment by UlrichElling** [comment]
We found another one. Also I tried to illustrate a bit what the mutations are like:

**Comment by UlrichElling** [comment]
![Screenshot 2022-07-15 at 15 08 29](https://user-images.githubusercontent.com/109073004/179229430-a23ccc96-3a6c-49ab-afd8-e9a23fbdec01.png)

**Comment by FedeGueli** [comment]
23 sequences as today 
Early signs of growth advantage vs BA.5 limited to Germany:
https://cov-spectrum.org/explore/Germany/AllSamples/Past3M/variants?aaMutations=M%3A3N&nucMutations=12160A&variantQuery1=%5B4-of%3AC4927T%2CC7834T%2CC12049T%2CA22001G%2CG22599A%2CT22942G%2CC24863T%2CA27507G%5D&analysisMode=CompareToBaseline&

@chrisruis @corneliusroemer

**Comment by shay671** [comment]
27 samples as of yesterday

https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_3a8c3_7e2120.json?c=pango_lineage_usher&label=nuc%20mutations:C15240T,A22001G,G22599A,C24863T,A27507G

[GISAID ref.csv](https://github.com/cov-lineages/pango-designation/files/9149227/GISAID.ref.csv)

**Comment by UlrichElling** [comment]
With the search term Spike_K147E, Spike_N460K, Spike_G339D, Spike_H69del currently a total of 60 cases are found. We mostly sequence only spike and selected regions in Austria, so these samples are only found with a term that does not demand too many mutations beyond spike. Could we maybe adapt the search term to account for that? 2 samples were also full-genome sequenced already. Really looking like growth advantage. Plus the spread across Germany and beyond says it all...
![image](https://user-images.githubusercontent.com/109073004/180005968-d957dfe5-c284-47fc-aa58-67036b0ec2a8.png)

**Comment by corneliusroemer** [comment]
@FedeGueli I would recommend using advanced query with `NextcladePangoLineage:BA.5*` instead of just two mutations which do not immediately show which lineage you compare to. I've edited your [growth advantage query](https://cov-spectrum.org/explore/Germany/AllSamples/from=2022-05-11&to=2022-07-20/variants?variantQuery=nextcladePangoLineage%3ABA.5*&variantQuery1=%5B4-of%3AC4927T%2CC7834T%2CC12049T%2CA22001G%2CG22599A%2CT22942G%2CC24863T%2CA27507G%5D&analysisMode=CompareToBaseline&)

**Comment by corneliusroemer** [comment]
Here's a sc2rf pic with donors BA.1 and BA.2. I tried to find which branch the BA.2* donor is from but couldn't find anything interesting in Germany/Europe. For the BA.1 part, there are no private mutations so if anything one can go by exclusion principle.

<img width="2195" alt="image" src="https://user-images.githubusercontent.com/25161793/180064866-cf0d88ca-9c77-47ed-a030-d0419afc3681.png">

**Comment by JosetteSchoenma** [comment]
Thanks, @corneliusroemer . Somehow I forgot to include it. Couldn't have done my analysis without it. Here is the one I made from the first 16 sequences. It won't be very different.
![image](https://user-images.githubusercontent.com/94831035/180068034-2dc94420-2149-4b9b-9ded-0bdaa539863e.png)

**Comment by corneliusroemer** [comment]
@chrisruis @InfrPopGen @thomasppeacock 

There are now at least 55 sequences of this recombinant, 37 have been submitted since Tuesday!

I would strongly recommend quick designation as the criteria are satisified, the Spike profile is very interesting and there appears to be growth, too.

Here are recent EPI_ISLs of complete sequences (3 are Spike only) so you can designate even if these sequences aren't yet in the Usher tree (maybe excluded due to long branch? @AngieHinrichs 

```txt
EPI_ISL_13269123, EPI_ISL_13269776, EPI_ISL_13344896,
EPI_ISL_13380484, EPI_ISL_13382506, EPI_ISL_13382561,
EPI_ISL_13382835, EPI_ISL_13385893, EPI_ISL_13387015,
EPI_ISL_13387937, EPI_ISL_13393217, EPI_ISL_13567682,
EPI_ISL_13569385, EPI_ISL_13570913, EPI_ISL_13795661,
EPI_ISL_13910672, EPI_ISL_13911687, EPI_ISL_13913622,
EPI_ISL_13917987, EPI_ISL_13919465, EPI_ISL_13923147,
EPI_ISL_13923171, EPI_ISL_13923416, EPI_ISL_13923424,
EPI_ISL_13923524, EPI_ISL_13923983, EPI_ISL_13924147,
EPI_ISL_13927733, EPI_ISL_13933571, EPI_ISL_13933603,
EPI_ISL_13933607, EPI_ISL_13933624, EPI_ISL_13933724,
EPI_ISL_13935440, EPI_ISL_13936383, EPI_ISL_13936388,
EPI_ISL_13936925, EPI_ISL_13937107, EPI_ISL_13937273,
EPI_ISL_13937536, EPI_ISL_13938846, EPI_ISL_13938960,
EPI_ISL_13939246, EPI_ISL_13939997, EPI_ISL_13941569,
EPI_ISL_13942052, EPI_ISL_13943854, EPI_ISL_13943861,
EPI_ISL_13944742, EPI_ISL_13945288, EPI_ISL_13952188-13952189,
```

**Comment by AngieHinrichs** [comment]
Thanks @corneliusroemer -- yep, they are filtered out due to having too many reversions relative to BA.2 placement by nextclade, but I've been manually exempting the EPI_ISL's from Josette's initial list and EPI_ISL_13719505 from the cov-spectrum query as of a few days ago.  They're in the tree, on a branch with the label proposed823 (Pango lineage assigned by UShER).

My list has a few that are not in your list:
EPI_ISL_13269235
EPI_ISL_13569221
EPI_ISL_13719505

I will exempt the rest from your list, so they should be added in tomorrow's build.

**Comment by corneliusroemer** [comment]
Thanks @AngieHinrichs! My list was created using manual AA substitution queries in GISAID - not super reliable. So not surprised you catch a few more.

**Comment by FedeGueli** [comment]
The @shay671 query finds now 63 sequences of this lineage:
https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?variantQuery=%5B4-of%3AC4927T%2CC7834T%2CC12049T%2CA22001G%2CG22599A%2CT22942G%2CC24863T%2CA27507G%5D&

**Comment by InfrPopGen** [comment]
Thanks for submitting. We've added recombinant lineage XAK with 55 newly designated sequences, and 0 updated designations

**Comment by FedeGueli** [comment]
This morning discussing about BA.3.2 we noticed that BA.3 (as BA.1) has no Orf1b:R1315C and  XAK lacks it too. i looked for that in this issue but i couldn't find it was discussed before and also it is missing from the Shay's list . i leave this here .
https://nextstrain.org/fetch/genome.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice1_genome_3efb15_5199d0.json?label=id:node_3116848
Maybe it is in some hidden comments, in the case i apologize
